README.md

```diff
@@ -1,24 +1,25 @@
 # MeesGame
 The something and only game without a number!
-This is gonna "git" gud
+
+## Disclaimer
+This is an educational software creation project, issued by University Utrecht. 
+This product serves no purpose other than educational, and, as such, should not be regarded as anything but educational. 
+All game visuals were either built within Microsoft Visual Studio or hand-drawn with image editing software GIMP.
+All sounds originated from the web sites www.soundjay.com, www.freesoundeffects.com, and videos from www.youtube.com,
+each with their respective Creative Commons Attribution license (re-usage allowed). For the background music, the theme song
+from the TV show "Prison Break" is used, which also has a Creative Commoms Attribution license.
+
 
 ### Dependencies
  - For building the release with an installer, install [NSIS](http://nsis.sourceforge.net/Download) in `C:\Program Files (x86)\NSIS`
+ 
+ 
+### System requirements
+ - Intel Core© 2 Duo T6670 2.2 GHz (Dual-core) CPU
+ - AMD Radeon 3470M GPU or Intel HD Graphics
+ - 1 GB of available disk space (minimal), 2GB of available disk space (recommended)
```

Comment:
Uhm yeah, so I thought the size was ~300 MB but that was due to a bug which caused the installer to include itself, which meant the size was doubled every time I ran it. That also explains why building took more than 5 minutes... The real installer size is 30 MB and the program files dir size is 60 MB, so we could probably reduce this by a factor of 10...